src/trackPromise.js

```diff
@@ -43,5 +43,24 @@ const decrementCounter = area => {
   counter[area]--;
 };
 
+export const manuallyResetPromiseCounter = area => {
+  area = area || defaultArea;
+  counter[area] = 0;
+  emitter.emit(promiseCounterUpdateEventId, false, area);
+};
+
+export const manuallyDecrementPromiseCounter = area => {
+  area = area || defaultArea;
+  decrementPromiseCounter(area);
+};
+
+export const manuallyIncrementPromiseCounter = area => {
+  area = area || defaultArea;
+  incrementCounter(area);
+
+  const promiseInProgress = anyPromiseInProgress(area);
```

Comment:
We could create `incrementPromiseCounter` method as we have with `decrement` and use it in 15-18 lines and here. What do you think?

Reply:
Great. I'm doing it.

Reply:
Done.